src/openfermion/utils/_unitary_cc_test.py

```diff
@@ -0,0 +1,204 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""Module to test unitary coupled cluster operators."""
+
+from __future__ import absolute_import
+
+import numpy
+import os
+import scipy
+import unittest
+
+from numpy.random import randn
```

Comment:
one space before the openfermion imports.

Reply:
k